test/unit/b2http/test_b2http.py

```diff
@@ -117,7 +118,7 @@ def test_invalid_json(self):
         response.content = b'{' * 500
         response.url = 'https://example.com'
 
-        with self.assertRaises(InvalidJsonResponse) as error:
+        with self.assertRaises(BadRequest) as error:
             B2Http._translate_errors(lambda: response)
 
             content_length = min(len(response.content), len(error.content))
```

Comment:
It was already this way before your changes, but shouldn't `content_length = ...` and the following line be indented back? Unless I'm seriously missing something, `B2Http._translate_errors(...)` will raise `BadRequest`, it will be captured as `error`,  and `content_length = ...` line will never be executed.

Reply:
you are absolutely correct :)